File: ruby/arrays/functions_operating_on_arrays/operations_on_elements/push.rb
#!/usr/bin/ruby2.7

array = [2, 4, 6]

puts("Initial state:\n\n")
puts(array.inspect, "\n")

r = array.push(8)

puts("After append by array.push(8):\n\n")
puts(array.inspect, "\n")
puts("Returned value: #{r}\n\n")

r = array.push(10, 12, 14)

puts("After append by array.push(10, 12, 14):\n\n")
puts(array.inspect, "\n")
puts("Returned value: #{r}\n\n")

File: javascript/arrays/functions_operating_on_arrays/operations_on_elements/push.js
/*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 */

let array = [2, 4, 6];

console.log("Initial state:\n");
console.log(array); console.log();

let r = array.push(8);

console.log("After append by array.push(8):\n");
console.log(array); console.log();
console.log(`Returned value: ${r}\n`);

r = array.push(10, 12, 14);

console.log("After append by array.push(10, 12, 14):\n");
console.log(array); console.log();
console.log(`Returned value: ${r}\n`);
